packages/oidc/src/refresh/refreshGrant.spec.ts

```diff
@@ -63,16 +63,20 @@ describe("refreshGrant", () => {
     );
   });
 
-  it("does not use basic auth if no client secret is available", async () => {
+  it("include the client id in the body if no client secret is available", async () => {
     const myFetch = mockFetch(JSON.stringify(mockBearerTokens()), 200);
     const client = mockClient();
 
     await refresh("some refresh token", mockIssuer(), client);
     expect(myFetch.mock.calls[0][0]).toBe(
```

Comment:
It would be a good idea to store `myFetch.mock.calls[0]` in a constant, refreshFetchArgs, and then reuse that in subsequent assertions; when I initially read this I thought lines 75 & 79 tested on a different fetch call, but then I remembered how mock captures args

Reply:
d218289054cd254c39a7c8c7d67cc568d2b5ecd1